third_party/nvidia/backend/compiler.py

```diff
@@ -316,21 +316,26 @@ def make_cubin(src, metadata, opt, capability):
             ]
             try:
                 subprocess.run(ptxas_cmd, check=True, close_fds=False, stderr=flog)
+                if os.path.exists(fsrc.name):
+                    os.remove(fsrc.name)
+                if os.path.exists(flog.name):
+                    os.remove(flog.name)
             except subprocess.CalledProcessError as e:
                 with open(flog.name) as log_file:
                     log = log_file.read()
+                if os.path.exists(flog.name):
+                    os.remove(flog.name)
+
                 if e.returncode == 255:
-                    raise RuntimeError(f'Internal Triton PTX codegen error: \n{log}')
+                    error = 'Internal Triton PTX codegen error'
                 elif e.returncode == 128 + signal.SIGSEGV:
-                    raise RuntimeError(
-                        f'Please run `ptxas {fsrc.name}` to confirm that this is a bug in `ptxas`\n{log}')
+                    error = '`ptxas` raised SIGSEGV'
                 else:
-                    raise RuntimeError(f'`ptxas` failed with error code {e.returncode}: \n{log}')
-            finally:
-                if os.path.exists(fsrc.name):
-                    os.remove(fsrc.name)
-                if os.path.exists(flog.name):
-                    os.remove(flog.name)
+                    error = f'`ptxas` failed with error code {e.returncode}'
+
+                raise RuntimeError(f'{error}\n'
+                                   f'`ptxas` stderr:\n{log}\n'
+                                   f'Repro command: {cmd}\n')
```

Comment:
Is cmd defined somewhere?

Reply:
Good catch! That variable got renamed after rebasing, let me fix it.